nn_sdwan/assets/monitors/high-latency-monitor.json

```
@@ -0,0 +1,23 @@
+{
+    "id": 65951821,
+    "name": "High Latency Alert [>100ms]",
```

Should be name of integration 
```suggestion
    "name": "nn_sdwan",
```

---

If I change this then the monitor itself would be named "nn_sdwan" when a customer/user is browsing their monitor list. They wouldn't know what it actually does or alerts on, unless they go inside and inspect the configuration.

Is that not what would happen here? If so, seems counterintuitive.